src/ble/BleConfig.h

```diff
@@ -207,6 +197,25 @@
 #define BLE_CONFIG_ERROR(e) (BLE_CONFIG_ERROR_MIN + (e))
 #endif // BLE_CONFIG_ERROR
 
+
+/**
+ * @def BTP_CONN_RSP_TIMEOUT
+ *
+ * @brief
+ *   Maximum amount of time, in milliseconds, after sending or receiving a BTP Session Handshake
+ *   request to wait for connection establishment.
+ */
+#define BTP_CONN_RSP_TIMEOUT 15000 // 15 seconds
```

Comment:
In case someone wonders why it doesn't match the spec, see https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/5710.

Reply:
it would be nice if constant names would use a suffix to indicate unit, like `BTP_CONN_RSP_TIMEOUT_MS` and `BTP_ACK_TIMEOUT_MS`

Reply:
The 15s was due to PBKDF being done at runtime during processing of PBKDFParamRequest, back before devices stored verifiers. I think the 5 seconds should actually be done in SDK, since we no longer do PBKDF during message processing.

Reply:
Added units in the constant names.
@tcarmelveilleux I see, thanks for clarification.